bundle-httpcache/src/main/java/com/adobe/acs/commons/httpcache/store/mem/impl/MemHttpCacheStoreImpl.java

```diff
@@ -214,7 +268,7 @@ public TabularData getCacheStats() throws OpenDataException {
                 .LONG});
 
         final TabularDataSupport tabularData = new TabularDataSupport(new TabularType("cacheEntries", "Cache " +
-                "Entries", cacheEntryType, new String[]{}));
+                "Entries", cacheEntryType, new String[]{"averageLoadPenalty"}));
```

Comment:
The last param cannot be empty. I think we want to flip this TabularData eventually. IIRC this last String[] are the set of cols that define unique row in the table (which in this case would be all cols?). Can revist later; wanted to get the MBean displaying atleast.